faker.go

```diff
@@ -766,10 +766,7 @@ func extractStringFromTag(tag string) (interface{}, error) {
 		if len(lenParts) != 2 {
 			return nil, fmt.Errorf(ErrWrongFormattedTag, tag)
 		}
-		strlen, err = strconv.Atoi(lenParts[1])
-		if err != nil {
-			return nil, fmt.Errorf(ErrWrongFormattedTag, tag)
-		}
+		strlen, _ = strconv.Atoi(lenParts[1])
```

Comment:
why ignore the error again?

Reply:
in this concrete case lenParts[1] are always some digits, because of the regex `len=\d+`. But if you think that we should check error anyway, because, for example, someone can change this regex, or smth else can happen, I'll do it